### DISCUSSION TITLE
Documentation source code

### DISCUSSION THREAD

**Discussion by Mister-Hope** [dotnetcore/FreeSql#1949] [Q&A]
I would like to help you maintain documentation site, but where are the docuementation source code?

It should be under docs folder in the past.

**Comment by 2881099** [comment]
| Documenttion website | Source code | Description |
| -- | -- | -- |
| 1. https://freesql.net | https://github.com/2881099/FreeSql.Wiki.VuePress | At the bottom of each detail page, there is an editing link |
| 2. https://github.com/dotnetcore/FreeSql/wiki | Github wiki |  |
